SofaKernel/modules/Sofa.Config/src/sofa/config.h.in

```diff
@@ -196,6 +196,11 @@ typedef unsigned __int64	uint64_t;
         "ExecParams are not needed anymore. " \
         "Remove the ExecParams argument from your call. ")
 
+#define SOFA_ATTRIBUTE_DISABLED__BASEOBJECT_SEARCH(msg) \
+    SOFA_ATTRIBUTE_DISABLED( \
+        "v20.06 (PR#1842)", "v21.06 (PR#1842)", \
```

Comment:
I think it should be `"v21.06 (PR#1842)", "v21.06 (PR#1842)"` as there is actually no deprecation period.